napari/__init__.py

```diff
@@ -1,16 +1,28 @@
-import warnings
+import os
+from pathlib import Path
+from qtpy import API_NAME
 
-vispy_warning = "VisPy is not yet compatible with matplotlib 2.2+"
+if API_NAME == 'PySide2':
+    # Set plugin path appropriately if using PySide2
```

Comment:
this should specify that this is merely a bugfix for when _both_ pyqt and pyside are installed

Reply:
done